### ISSUE TITLE
cannot import name 'LonaApp' from 'lona'

### ISSUE THREAD

**Issue by sylvoslee** [lona-web-org/lona#187]
`
Traceback (most recent call last):
  File "test_lona.py", line 2, in <module>
    from lona import LonaView, LonaApp
ImportError: cannot import name 'LonaApp' from 'lona'
`

**Comment by fscherf** [comment]
Hi @sylvoslee,

which Lona version are you using? How did you install Lona?

**Comment by sylvoslee** [comment]
lona 1.8.4 on windows
install by `pip install lona`

**Comment by fscherf** [comment]
@sylvoslee does ``import lona`` work? If not, i suspect the script runs in a different environment than your pip command.
Did this work before?

**Comment by sylvoslee** [comment]
`>>> import lona
>>> dir(lona)
['ClientError', 'ForbiddenError', 'LonaView', 'MATCH_ALL', 'NotFoundError', 'Route', 'ServerStop', 'StopReason', 'UserAbort', 'VERSION', 'VERSION_STRING', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', '_json', 'command_line', 'connection', 'default_settings', 'errors', 'events', 'exceptions', 'html', 'imports', 'protocol', 'request', 'routing', 'shell', 'static_files', 'unique_ids', 'view', 'view_runtime', 'worker_pool']
>>>`

**Comment by fscherf** [comment]
Huh!? Interesting! That seems to be some packaging or importing problem.

Does ``from lona.app import LonaApp`` work? 

In ``lona/__init__.py`` is a try Block around the imports. What does happen if you remove it?

I dont have access to a windows machine. Can someone reproduce this problem? @korantu @themutt @sobolevn

**Comment by themutt** [comment]
I checked on my VM with windows 10 and python 3.10. The error seems to be related to the fact the `lona` is trying to import the [syslog](https://docs.python.org/3/library/syslog.html) package but syslogd is only available on Unix.
See the traceback here:
```
Traceback (most recent call last):
  File "c:\users\grigolet\downloads\lonatest2\lona\lona\__init__.py", line 6, in <module>
    from .app import LonaApp
  File "c:\users\grigolet\downloads\lonatest2\lona\lona\app.py", line 15, in <module>
    from lona.command_line.run_server import run_server
  File "c:\users\grigolet\downloads\lonatest2\lona\lona\command_line\run_server.py", line 12, in <module>
    from lona.logging import setup_logging
  File "c:\users\grigolet\downloads\lonatest2\lona\lona\logging.py", line 6, in <module>
    import syslog
ModuleNotFoundError: No module named 'syslog'
```
I am not sure what can be done about it. What do you think @fscherf ?

**Comment by fscherf** [comment]
@themutt Oh! You are right. I forgot that syslog is unix specific. I will create a pull request that changes the code so that the logging checks if syslog can be imported, and if not, this feature gets disabled.

Thank you for testing! Can you test my pr as well? I think I should setup an vm like that too

**Comment by themutt** [comment]
@fscherf Sure! No problem :+1: When I was looking for the stack trace I noticed that in `__init__.py` eventual import errors are simply catched and passed.
https://github.com/lona-web-org/lona/blob/dcaac17c396c25f3812c92802d8905eb7b1fd2e6/lona/__init__.py#L8-L11

 Wouldn't it be better to print anyway the exception?

**Comment by fscherf** [comment]
@themutt i though of that too. Good idea, i will add that to the pull-request

**Comment by fscherf** [comment]
@sylvoslee: I released 1.8.5 which has a fix for this problem. Thanks for reporting! :)
